documentation/AuthorizationService.md

```diff
@@ -1,9 +1,9 @@
 ## Authorization Service
 
-The authorization service provides API that allows to add and retrieve authorization data for GPII App installations. GPII App Installations are authorized using [Resource Owner GPII Token Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Resource_Owner_GPII_Token_Grant).
+The authorization service provides API that allows to add and retrieve authorization data for GPII App installations. GPII App Installations are authorized using [Resource Owner GPII key Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Resource_Owner_GPII_Key_Grant).
 
 * [getAuthorizationByAccessToken(accessToken)](#getauthorizationbyaccesstokenaccesstoken)
-* [grantGpiiAppInstallationAuthorization(gpiiToken, clientId)](#grantgpiiappinstallationauthorizationgpiitoken-clientid)
+* [grantGpiiAppInstallationAuthorization(gpiiKey, clientId, clientCredentialId)](#grantgpiiappinstallationauthorizationgpiikey-clientid-clientCredentialId)
```

Comment:
Link target should be `#grantgpiiappinstallationauthorizationgpiikey-clientid-clientcredentialid` -- all lowercase.